tests/test_syntax_errors.py

```diff
@@ -92,3 +92,28 @@ def test_sets(self):
                    '    a:\n'
                    '      set\n'
                    '...\n', None)
+
+    def test_invalid_char_last(self):
+        self.check('---\nkey: value\t\n',
```

Comment:
Can you use the more-readable format that is used everywhere is tests?
```suggestion
        self.check('---\n'
                   'key: value\t\n',
```